dist/tools/jlink/jlink.sh

```diff
@@ -58,7 +58,12 @@ _JLINK_SERVER=JLinkGDBServer
 _JLINK_IF=SWD
 _JLINK_SPEED=2000
 # default starting address of the devices flash memory
-_FLASH_ADDR=0
+if [ -n "${FLASH_OFFSET}" ]; then
+    _FLASH_ADDR=${FLASH_OFFSET}
+    echo "Flashing at offset ${FLASH_OFFSET}"
+else
+    _FLASH_ADDR=0
+fi
```

Comment:
Just to complete this... `JLINK_FLASH_ADDR` is used only for the Nordic's BLE softdevice, which doesn't work since a longtime... So removing that variable AND the broken support for softdevice would be cleaner.